### ISSUE TITLE
feature request: select presence from the list

### ISSUE THREAD

**Issue by vrobolab** [chrm-software/cryptmee#4]
Currently I have to type the contact's presence manually to send the message to desired contact's device, which is not very convinient. 
I'd like to see a list of contact's active presences to choose from, with priority number next to the presence name.

**Comment by chrm-software** [comment]
Do you mean the xmpp resource? In fact, I have no idea if this really works. Testing it with two different devices (configured for the same JID, like user@server/device1 and user@server/device2) shows messages always on both devices. I'm afraid the meego xmpp stack is just cutting the resource information :(

**Comment by vrobolab** [comment]
Yes I mean the xmpp resource, ability to select the device to which send the message.
